### PR TITLE
7522 fix modal window

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam could you take a look?

Change to SVG
image

Change to SVG and fix overflow
image

Add Icons and visual fixes (thanks @nobuti)
image

### CODE REVIEW

**Comment by nobuti** [comment]
@piensaenpixel there is a thing that perturbs me a little. The selected button has a 2px border, it changes the vertical alignment with the rest of the buttons. Perhaps we can change it to use box-shadow instead border.

**Comment by xavijam** [comment]
Could we remove sketch comments from SVGs?

File: app/assets/stylesheets/editor-3/widget-list.css.scss [hunk 1]
```diff
@@ -69,4 +53,54 @@
...
+}
+.Editor-navButtonsItem.is-selected .WidgetIcon--category {
+  background-image: image-url('layout/widgets/category_color.svg');
+}
+
+.WidgetIcon--formula {
+  background-image: image-url('layout/widgets/formula_bn.svg');
+
+}
+.Editor-navButtonsItem.is-selected .WidgetIcon--formula {
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/widget-list.css.scss]: with `.is-selected .WidgetIcon--formula` is not enough?

Same question for previous things.

File: lib/sql [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 58fd5d4060d02276603a7180b85317a02a9fb3e5
```
Comment (xavijam) [inline lib/sql]: I feel we don't need this, please, checkout it :).

**Comment by xavijam** [comment]
Apart from the sql submodule thing and two minor things, 👍, no much to say, you both are :magic-wand:

File: app/assets/stylesheets/editor-3/widget-list.css.scss (hunk 1 above)
Comment (piensaenpixel) [inline app/assets/stylesheets/editor-3/widget-list.css.scss]: because maybe in some parent we could find is-selected... only for avoid conflicts

**Comment by nobuti** [comment]
FYI I use this to clean the svg's crap: https://gist.github.com/bomberstudios/983e2ed8613515e97e5d

**Comment by piensaenpixel** [comment]
Fix PR, could you take a look @xavijam @nobuti ?

File: app/assets/stylesheets/editor-3/modals.css.scss [hunk 3]
```diff
@@ -183,9 +183,10 @@
 }
 
 .Editor-navButtonsItem.is-selected .Editor-navButtonsItemLink {
-  border: 2px solid $cBlue;
+  border: 1px solid $cBlue;
   background: $cWhite;
   color: $cMainBg;
+  box-shadow: 1px 0 0 $cBlue, -1px 0 0 $cBlue, 0 1px 0 $cBlue, 0 -1px 0 $cBlue;
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/modals.css.scss]: why not `box-shadow: 0 0 0 1px $cBlue;`? Just curiosity.

**Comment by piensaenpixel** [comment]
it this ok for merge? :)

**Comment by nobuti** [comment]
scary-yes

**Pull request merged**